feat: add scopedb logo and update layout and component styles

hi-caicai opened the pull request:
- Added new scopedb logo image.
- Adjusted main padding in layout for better spacing.
- Enhanced DesktopTableOfContents with ConfigProvider for Anchor component styling.
- Updated Header and MobileNav to use the new logo and adjusted text styles.
- Improved Sidebar component styles for better theme integration.

tisonkun commented on website/src/overrides/PageFrame.astro:
```diff
@@ -57,7 +59,7 @@ const { hasSidebar } = Astro.locals.starlightRoute;
 
   .sidebar-pane {
```
Typo of "panel"?

STRRL replied:
No, that‘s intentional.

ref:  https://github.com/withastro/starlight/blob/9bb064ef4228befd809499c3f26553190e2a2079/packages/starlight/components/PageFrame.astro#L50

btw, as we're going to redesign the styling with tailwindcss, it might be totally replaced with other styling in the future.

tisonkun commented:
Pending for replacing as the new main branch.

hi-caicai committed:
feat: add scopedb logo and update layout and component styles

- Added new scopedb logo image.
- Adjusted main padding in layout for better spacing.
- Enhanced DesktopTableOfContents with ConfigProvider for Anchor component styling.
- Updated Header and MobileNav to use the new logo and adjusted text styles.
- Improved Sidebar component styles for better theme integration.

tisonkun commented on src/components/Header/MobileNav.tsx:
```diff
@@ -164,7 +164,7 @@ export default function MobileNav() {
                     <div className="flex items-center justify-between px-[12px] py-[14px] border-b border-[rgba(0,0,0,0.06)] bg-[#fff]">
                         <div id="mobile-nav-logo" className="flex items-center gap-3">
-                            <Image src="/logo-banner.svg" alt="ScopeDB Logo" height={36} width={128} />
+                            <Image src="/scopedb-logo.png" alt="ScopeDB Logo" height={32} width={120} />
```
What's the difference between a SVG file and a PNG file?

hi-caicai replied:
Maintain a consistent format, size, and images with the site

tisonkun replied:
OK. I don't have a strong feeling here. If both is fine I'd accept this change ;D

tisonkun commented on src/components/Sidebar/index.tsx:
```diff
@@ -100,7 +123,7 @@ export default function Sidebar({ prefix, className }: { prefix: string, classNa
             openKeys={stateOpenKeys}
             onOpenChange={onOpenChange}
-            className={className}
+            className={`sidebar-container ${className || ''}`}
```
There should be no longer a class named "sidebar-container". I think this line can retain:

```suggestion
            className={className}
```

?

tisonkun replied:
Or `className={clsx(className)}`

hi-caicai committed:
Update src/components/Sidebar/index.tsx

Co-authored-by: tison <wander4096@gmail.com>

tisonkun reviewed (approved):
Thanks for your follow-up! Merging ...
